docs/modules/operation/pages/events/sources/eif-adapter.adoc

```diff
@@ -1,9 +1,9 @@
 
 [[ga-events-sources-eif]]
-= Receive IBM Tivoli Event Integration Facility Events
+= IBM Tivoli Event Integration Facility
 
 You can configure {page-component-title} to receive events sent using the https://www.ibm.com/docs/en/netcoolomnibus/8.1?topic=reference-overview-tivoli-event-integration-facility[Tivoli Event Integration Facility].
-These EIF events are translated into OpenNMS events using preconfigured rules.
+These EIF events are translated into {page-component-title} events using preconfigured rules.
 The resulting UEI are anchored in the `uei.opennms.org/vendor/IBM/EIF/` namespace, with the name of the EIF https://www.ibm.com/support/knowledgecenter/SSSHTQ_8.1.0/com.ibm.netcool_OMNIbus.doc_8.1.0/omnibus/wip/eifsdk/concept/ecoemst16.html[event class] appended.
```

Comment:
"event class" link goes to a content not found page.

Reply:
removing link.  If you use EIF, you can find the class for your current version.